pkg/downloader/custom_github_detector.go

```
@@ -0,0 +1,110 @@
+package downloader
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/charmbracelet/log"
```

golangci-lint (1.62.2)

import "github.com/charmbracelet/log" imported without alias but must be with alias "log" according to config

(importas)